x-pack/plugin/ml/qa/native-multi-node-tests/build.gradle

```diff
@@ -27,6 +27,9 @@ task copyKeyCerts(type: Copy) {
 sourceSets.test.resources.srcDir(keystoreDir)
 processTestResources.dependsOn(copyKeyCerts)
 
+// Disabled and tracked here https://github.com/elastic/elasticsearch/issues/45405
+integTests.enabled = false
```

Comment:
A couple things here:

1. This will error because there is no task named `integTests`. The it should be `integTest.enabled = false`.
2. Disabling the test task will cause the testing conventions task to fail so we'll need to temporarily disable that as well with `testingConventions.enabled = false`.

Reply:
Didn't realize this PR was merged. I've fixed this in 8ed84bc5cef9bcfae6c817059f764d97e4451a4a